demo/html-spec-native-lapi/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+</head>
+
+<body>
+
+  <html-spec-viewer></html-spec-viewer>
+
+  <p id="streamRequirement">This demo requires
+    <a href="https://www.chromestatus.com/feature/5466425791610880">Streams API: TransformStream</a> support.
+  </p>
+
+  <div id="lapiRequirement">This demo uses the built-in virtual-list from the Layered API. Enable one of the following flags in chrome://flags to try
```

Comment:
I'd say: "If you are using Chrome, ensure you are using version 68.0.3420.0 or greater, and enable one..."

Other browsers might have different instructions.